### PR TITLE
Implement transaction fee breakdown with tooltip

### LINKED ISSUE #4808: Implement transaction fee breakdown with tooltip
### Description

Enable users to view the transaction fee breakdown and provide tooltip to understand each type of fee charged.

### Acceptance Criteria

- Implement the [fee breakdown](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?node-id=3717-256433&t=WoICiuwIoJti7tEL-4) as designed
- Implement [tooltip](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?node-id=3244%3A262685&t=honCttpq7xCrhLqN-4) as designed
- Ensure both dark and light mode are implemented
- Add tests to check the fee breakdown scenario
- Add tests to check and match the tooltip

### PR DESCRIPTION
**Pull request by ikem-legend**
### What was the problem?

This PR resolves #4808

### How was it solved?

- Implementing a tooltip for the fee breakdown details

### How was it tested?

- Visually

### CODE REVIEW

File: src/modules/transaction/components/TransactionPriority/TransactionPriority.css [hunk 1]
```diff
@@ -83,6 +83,29 @@
 
       & .fieldLabel {
         display: inline-block;
+
+        & div > div {
+          width: 18vw !important; /* stylelint-disable-line */
```
Comment (oskarleonard) [inline src/modules/transaction/components/TransactionPriority/TransactionPriority.css]: image

probably need 16vw.
Also, no need for !important

File: src/modules/transaction/components/TransactionPriority/TransactionPriority.css [hunk 2]
```diff
@@ -83,6 +83,29 @@
...
+              color: var(--color-zodiac-blue);
+            }
+
+            & .feesDetails {
+              color: var(--color-slate-gray);
+            }
+          }
+
+          @media (--large-viewport) {
+            width: 25vw !important; /* stylelint-disable-line */
```
Comment (oskarleonard) [inline src/modules/transaction/components/TransactionPriority/TransactionPriority.css]: no need for !important

**Review by ManuGowda — CHANGES_REQUESTED** [review]
- I think the first acceptance criteria is missed?
Implement the [fee breakdown](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?node-id=3717-256433&t=WoICiuwIoJti7tEL-4) as designed
- the second one is implemented but has some bugs with tooltip overlap

Screenshot 2023-03-23 at 10 27 13 AM
Screenshot 2023-03-23 at 10 27 33 AM

File: src/modules/transaction/components/TransactionPriority/TransactionPriority.css (hunk 1 above)
Comment (ikem-legend) [reply src/modules/transaction/components/TransactionPriority/TransactionPriority.css]: @oskarleonard Fixed

File: src/modules/transaction/components/TransactionPriority/TransactionPriority.css (hunk 2 above)
Comment (ikem-legend) [reply src/modules/transaction/components/TransactionPriority/TransactionPriority.css]: @oskarleonard Fixed

**Review by ManuGowda — COMMENTED** [review]
Fee breakdown doesn't handle all the `composedFeeList` value i.e initialisation fee, see attachment 

Screenshot 2023-03-23 at 4 38 57 PM

File: src/modules/transaction/components/TransactionPriority/FeeViewer.js [hunk 3]
```diff
@@ -93,10 +97,26 @@ const FeesViewer = ({
...
+            {title === 'Transaction' && (
+              <Tooltip position="top left">
+                <div className={styles.feesBreakdownRow}>
+                  <p>Fee breakdown</p>
+                  {transactionFeeList.map(({ type, value: feeValueInfo }, index) => (
+                    <Fragment key={`${index}-${type}-${feeValueInfo}`}>
+                      <span>{type.replace('Fee', '')}</span>
+                      <span>
+                        {Number(convertFromBaseDenom(feeValueInfo)).toFixed(6)} {token.symbol}
+                      </span>
```
Comment (eniolam1000752) [inline src/modules/transaction/components/TransactionPriority/FeeViewer.js]: I guess for the purpose of readability, you can abstract this component then consume it here which would avoid using `Fragment` here and also make it readable.

**Review by oskarleonard — DISMISSED** [review]

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]

File: src/modules/transaction/components/TransactionPriority/FeeViewer.js (hunk 3 above)
Comment (ikem-legend) [reply src/modules/transaction/components/TransactionPriority/FeeViewer.js]: @eniolam1000752 Fixed

**Review by eniolam1000752 — DISMISSED** [review]

**Review by ManuGowda — DISMISSED** [review]
Screenshot 2023-03-24 at 4 19 12 PM


For better user experience can you omit the decimals when its `0`? this way its easier to read `10 LSK`? if it was `10.0001 LSK` in understand we need to preserve the decimals, for this case we can omit it?.

**Review by oskarleonard — DISMISSED** [review]

**Comment by ikem-legend** [comment]
> Screenshot 2023-03-24 at 4 19 12 PM
> 
> For better user experience can you omit the decimals when its `0`? this way its easier to read `10 LSK`? if it was `10.0001 LSK` in understand we need to preserve the decimals, for this case we can omit it?.

Fixed

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**